### ISSUE TITLE
Issue with scons based installation getting mismatch arguments in getLineBand

### ISSUE THREAD

**Issue by ayush12gupta** [isce-framework/isce2#684]
When I am trying to install isce2 using Scons method i am getting this error that getLineBand are receiving different types of arguments i.e. complex and integer. I tried using the release v2.6.2 zip file for doing the installation. Can someone suggest if its issue with my fortran version or something else. 

![image](https://user-images.githubusercontent.com/56562280/227714485-90297e74-72e7-491b-9d71-6e0d7359744f.png)

**Comment by rtburns-jpl** [comment]
This is an issue with legacy fortran code in that module that doesn't work with recent versions of gfortran. Try using gfortran 9 or older.

**Comment by ayush12gupta** [comment]
Ok thanks it got resolved.

**Comment by ayush12gupta** [comment]
After successful installation some of the packages are not working, when i tried to imort TopsProc i am getting this error.

![image](https://user-images.githubusercontent.com/56562280/227770155-621e0de8-c4e1-4c56-8ad7-2e7bb632722d.png)

**Comment by ayush12gupta** [comment]
@rtburns-jpl can you please suggest if there is some mistake in installation from my side and so that how can i respose it.

**Comment by rtburns-jpl** [comment]
Did your cmake detect cython properly? That's the only thing I can think of that would result in the alos2proc_f module not being built

**Comment by ayush12gupta** [comment]
I installed it using scons installation method and yes initially it detected all the required packages

**Comment by ayush12gupta** [comment]
> Did your cmake detect cython properly? That's the only thing I can think of that would result in the alos2proc_f module not being built

I installed it using scons installation method and yes initially it detected all the required packages

**Comment by rtburns-jpl** [comment]
I'm not sure what happened then, I haven't seen that issue before. I'd suggest using the cmake build method and see if that works for you.

**Comment by ayush12gupta** [comment]
> I'm not sure what happened then, I haven't seen that issue before. I'd suggest using the cmake build method and see if that works for you.

Yeah I cmake works but i need to use the autoRIFT functionality which is not available with cmake installation.

**Comment by rtburns-jpl** [comment]
autorift works fine with cmake
